### PR TITLE
[Merged by Bors] - doc: Change old Lean 3 commands to Lean 4 in implementation notes

### PR DESCRIPTION
**Pull request by OmarMohsenGit**
I changed Lean's 3 old "variables" command to Lean's 4 command "variable" in some implementation notes.
I might have missed some

### CODE REVIEW

File: Mathlib/MeasureTheory/Function/L1Space.lean [hunk 1]
```diff
@@ -434,7 +434,7 @@ end NormedSpace
 /-! ### The predicate `Integrable` -/
 
 
--- variables [measurable_space β] [measurable_space γ] [measurable_space δ]
+-- variable [measurable_space β] [measurable_space γ] [measurable_space δ]
```
Comment (adomani) [inline Mathlib/MeasureTheory/Function/L1Space.lean:437]:
```suggestion
-- variable [MeasurableSpace β] [MeasurableSpace γ] [MeasurableSpace δ]
```
I think that this is the Mathlib4 spelling of these typeclasses.

**Comment by riccardobrasca** [comment]
Taking into account Damiano's comment LGTM, thanks!

bors d+

**Comment by riccardobrasca** [comment]
It seems like a random failure, let's try again.

bors merge

**Pull request closed without merging**